packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -25,29 +25,106 @@
 			</span>
 		</template>
 		<template v-if="datasetView === DatasetView.DESCRIPTION">
-			<section class="metadata data-row">
+			<div class="container">
+				<Message class="inline-message" icon="none"
+					>This page describes the dataset. Use the content switcher above to see the data table and
+					transformation tools.</Message
+				>
+			</div>
+			<section class="metadata data-row" v-if="!metadata">
 				<section>
+					<header>Rows</header>
+					<section>{{ csvContent?.length || '-' }}</section>
+				</section>
+				<section>
+					<header>Columns</header>
+					<section>{{ rawContent?.stats?.length || '-' }}</section>
 					<header>Metadata</header>
 					<section>{{ dataset?.metadata || '-' }}</section>
 				</section>
 				<section>
-					<header>URL</header>
+					<header>Date uploaded</header>
 					<section>
-						<a :href="dataset?.url">{{ dataset?.url || '-' }}</a>
+						{{ new Date(dataset?.timestamp as Date).toLocaleString('eu-ZA') || '-' }}
```

Comment:
What locale is this? Is this correct? I would have assumed en-US

Reply:
Damn @jamiewaese-uncharted that's not good, I will fix this.